ui/rewards/dashboard/RewardsInstancesModal.tsx

```diff
@@ -0,0 +1,98 @@
+import { Flex, Text, Box } from '@chakra-ui/react';
+import React from 'react';
+
+import type { RewardsInstance } from 'types/api/rewards';
+
+import useIsMobile from 'lib/hooks/useIsMobile';
+import { useColorModeValue } from 'toolkit/chakra/color-mode';
+import { DialogBody, DialogContent, DialogRoot, DialogHeader } from 'toolkit/chakra/dialog';
+import { Image } from 'toolkit/chakra/image';
+import { Link } from 'toolkit/chakra/link';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  items: Array<RewardsInstance> | undefined;
+};
+
+const RewardsInstancesModal = ({ isOpen, onClose, items }: Props) => {
+  const isMobile = useIsMobile();
+  const bgColor = useColorModeValue('blue.50', 'whiteAlpha.100');
+  const textColor = useColorModeValue('gray.700', 'whiteAlpha.800');
```

Comment:
```suggestion
  const bgColor = {_light: 'blue.50', _dark: 'whiteAlpha.100'};
  const textColor = {_light: 'gray.700', _dark: 'whiteAlpha.800'};
```

This style is preferable to `useColorModeValue` because it allows styles to be placed closer to the elements to which they are applied.